skbeam/core/accumulators/binned_statistic.py

```diff
@@ -130,7 +140,12 @@ def __init__(self, sample, statistic='mean',
         # Compute the bin number each sample falls into.
         Ncount = {}
         for i in np.arange(self.D):
-            Ncount[i] = np.digitize(sample[:, i], self.edges[i])
+            # Apply mask in a non-ideal way by setting value outside range.
+            # Would be better to do this using bincount "weights", perhaps.
+            thissample = sample[:, i]
+            thissample[mask == 0] = (self.edges[i][0] -
```

Comment:
Agreed.  I have added this in the most recent commit.  Also included @ordirules  suggestion of using radial_grid/angle_grid and updated phi-definition documentation accordingly.